pkg/cluster/kubeclient/syncpod.go

```diff
@@ -72,6 +72,7 @@ func (u *kubeclient) EnsureSyncPod(ctx context.Context, syncImage string, hash [
 					Spec: corev1.PodSpec{
 						Containers: []corev1.Container{
 							{
+								Args:            []string{"sync"},
```

Comment:
why not Command ?

Reply:
entrypoint = command in kubernetes 
https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes

Reply:
thanks!